workflow101.Rmd

```diff
@@ -234,6 +234,13 @@ To see the most useful keyboard shortcuts, press `Alt + Shift + K` or use *Help
 ::: callout-tip
 ## RStudio
 
+RStudio also provides the [Command Palette](https://docs.posit.co/ide/user/ide/reference/shortcuts.html#command-palette) which gives fast, searchable access to all of the IDE's commands -- especially helpful when you can't remember a particular keyboard shortcut.
```

Comment:
Aesthetically I don't love having three callout boxes in a row. Should this content go in the previous box with the keyboard shortcut tip?

Reply:
I'm sort of beyond this level of care 🥲  and I can get happy with whatever you decide.

Reply:
Ok, I merged them. I think it's cleaner.